client/v2/tx/cometbft.go

```diff
@@ -0,0 +1,36 @@
+package tx
+
+import (
+	"context"
+
+	rpcclient "github.com/cometbft/cometbft/rpc/client"
+	coretypes "github.com/cometbft/cometbft/rpc/core/types"
+)
+
+// CometRPC defines the interface of a CometBFT RPC client needed for
+// queries and transaction handling.
+type CometRPC interface {
```

Comment:
Can we delete this?

Reply:
Sure, I can move everything that is shared between V1 and V2 to a 'common' folder so it doesn't feel that cometbft.go (and several other files) is part of client/v1 nor v2 if that sounds good